assets/js/components/deposit/BasicInformation.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { CustomFields } from "react-invenio-forms";
+
+export class BasicCERNInformation extends Component {
+  render() {
+    const { customFieldsUI, record } = this.props;
+    return (
+      <CustomFields
+        config={customFieldsUI}
+        record={record}
+        templateLoaders={[
+          (widget) => import(`@templates/custom_fields/${widget}.js`),
```

Comment:
I was not aware of this, and I find it dangerous, as there is no build time error if the path does not exist.

This will bite us back one day.